src/components/migrationToV3.stories.mdx

```diff
@@ -85,6 +92,216 @@ const MyApp = () => {
+      <MyApp />
+    </MeetingProvider>
+  );
+};
+```
+
+change it to like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import { MeetingSessionConfiguration } from 'amazon-chime-sdk-js';
+
+const MyApp = () => {
+  const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+  meetingSessionConfiguration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = false;
+
+  const meetingManager = useMeetingManager();
+  await meetingManager.join(
+    meetingSessionConfiguration,
+  );
+
+  return(
+    <MeetingProvider>
+      <MyApp />
+    </MeetingProvider>
+  );
+};
+```
+
+##### enableWebAudio
+
+The following example uses `enableWebAudio` property as an example to showcase the change. `enableWebAudio` is chosen here because it is one of the properties that is affected and it is not part of `MeetingSessionConfiguration`. 
+
+If you code looks like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+
+const MyApp = () => {
+  const meetingConfig = {
+    enableWebAudio: true,
+  };
+
+  const meetingManager = useMeetingManager();
+  await meetingManger.join({
+    data.Meeting,
+    data.Attendee,
+  });
+
+  return(
+    <MeetingProvider {...meetingConfig}>
+      <MyApp />
+    </MeetingProvider>
+  );
+};
+```
+
+change it to like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import { MeetingSessionConfiguration } from 'amazon-chime-sdk-js';
+
+const MyApp = () => {
+  const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+  const options = {
+    enableWebAudio: true,
+  };
+
+  const meetingManager = useMeetingManager();
+  await meetingManager.join(
+    meetingSessionConfiguration,
+    options,
+  );
+
+  return(
+    <MeetingProvider>
+      <MyApp />
+    </MeetingProvider>
+  );
+};
+```
+
+### `MeetingManager` `join` method prop change
+
+In V3, `MeetingManager`'s `join` method takes `MeetingSessionConfiguraiton` and `options` as params.
+`meetingInfo` and `attendeeInfo` parameters are removed from the interface and are rather passed to the constructor of `MeetingSessionConfiguration`.
+
+If you code looks like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+
+const MyApp = () => {
+  const meetingManager = useMeetingManager();
+  await meetingManager.join({
+    data.MeetingInfo,
+    data.AttendeeInfo,
+  });
+};
+```
+
+change it to this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import { MeetingSessionConfiguration } from 'amazon-chime-sdk-js';
+
+const MyApp = () => {
+  const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+  
+  const meetingManager = useMeetingManager();
+  await meetingManager.join(
+    meetingSessionConfiguration,
+  );
+};
+```
```

Comment:
I would prefer removing this example. So many redundant examples.

Would prefer mentioning the `MeetingProvider props` change  and `meetingManager.join` API change at the same time in the same section. And use one example to cover all of them.

So the logic is
* we remove the part of props of `MeetingProvider` because we modify the `meetingManager.join` API to config that.